app/models/review.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: reviews
+#
+#  id            :integer          not null, primary key
+#  reviewer_name :string
+#  rating        :integer          not null
+#  comment       :text
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  restaurant_id :integer          not null
+#
+
+class Review < ApplicationRecord
+  belongs_to :restaurant
+  after_create :update_event_rating
+  validates_presence_of :rating, :restaurant
+
+  private
+
+  def update_event_rating
```

Comment:
Interesting that you chose to update the rating inside the review model. Any specific reason you chose to do it here rather than have a function inside the restaurant model which calculates the average?

Reply:
👍